dm/tests/shardddl4_1/run.sh

```diff
@@ -906,7 +907,7 @@ function DM_154 {
 function run() {
 	init_cluster
 	init_database
-	start=131
+	start=151
```

Comment:
need revert？

Reply:
addressed in https://github.com/pingcap/tiflow/pull/4087/commits/96b2db76737e4e6221b5f0413690e0be2d8a5e15